index.md

```diff
@@ -110,45 +117,50 @@ servo_one.position
 
 Note that this value is not in degrees!
 
-By default, servos will be un-powered when your robot starts, and can freely rotate. Once you set a value, they then fix to that rotation.
+By default, servos will be un-powered when your robot starts, and can freely rotate. Once you set a value, they then fix to that rotation. This also happens when you turn off your robot, or unplug your USB.
 
-## Cameras (Vision Boards)
-As always, a dictionary of the available cameras is obtained, then an individual camera is obtained using the serial number, or if there is only a single camera connected, then there is a function to get that as follows:
+### Ultrasound
+The ultrasound sensor can be used to measure the distance your robot is away from something.
 
+Assuming you've wired up the read (_echo_) and write (_trigger_) pins to 6 and 7:
```

Comment:
I'd recommend listing these in the order they're passed to the function (trigger before echo) - less chance of confusion that way